firefox_desktop/firefox_desktop.model.lkml

```diff
@@ -2,9 +2,9 @@ connection: "telemetry"
 
 include: "//looker-hub/firefox_desktop/views/*.view.lkml"
 include: "//looker-hub/firefox_desktop/explores/*.explore.lkml"
-include: "explores/*.explore.lkml"
-include: "views/*.view.lkml"
-# include: "dashboards/*.dashboard"
+include: "explores/*"
+include: "views/desktop_daily_retention.view.lkml"
```

Comment:
I believe this should still include `views/acquisition_funnel.view.lkml`, `views/client_counts.view.lkml`, and `views/urlbar_events_daily_table.view.lkml`, because otherwise with this change those view refinements are no longer being applied.
```suggestion
include: "views/acquisition_funnel.view.lkml"
include: "views/client_counts.view.lkml"
include: "views/desktop_daily_retention.view.lkml"
include: "views/urlbar_events_daily_table.view.lkml"
```